packages/model/src/manuscript.ts

```diff
@@ -588,6 +591,15 @@ export const manuscriptPostRequestSchema: JSONSchemaType<ManuscriptPostRequest>
     additionalProperties: false,
   };
 
+export const manuscriptPutRequestSchema: JSONSchemaType<ManuscriptPutRequest> =
+  {
+    type: 'object',
+    properties: {
+      status: { type: 'string', nullable: true },
```

Comment:
```suggestion
      status: { enum: manuscriptStatus, type: 'string' },
```
I think it should be required as well